src/coreclr/inc/corinfo.h

```diff
@@ -2494,9 +2494,8 @@ class ICorStaticInfo
 
     // returns the "NEW" helper optimized for "newCls."
     virtual CorInfoHelpFunc getNewHelper(
-            CORINFO_RESOLVED_TOKEN *    pResolvedToken,
-            CORINFO_METHOD_HANDLE       callerHandle,
-            bool *                      pHasSideEffects
+            CORINFO_CLASS_HANDLE  classHandle,
+            bool*                 pHasSideEffects
```

Comment:
Can you update the above documentation comment? It refers to "newCls", which doesn't exist (and didn't before, either), and doesn't describe `pHasSideEffects`.